lib/net.js

```diff
@@ -59,7 +59,7 @@ exports.createServer = function(options, connectionListener) {
 // connect(port, [host], [cb])
 // connect(path, [cb]);
 //
-exports.connect = exports.createConnection = function() {
+function connect() {
```

Comment:
What if we just called this `createConnection()` and added the `connect: createConnection` alias in the export list. This way we won't have to rename `connect` to `internalConnect` everywhere.